src/clusterfuzz/_internal/metrics/monitor.py

```diff
@@ -82,6 +82,9 @@ def __init__(self):
     self.daemon = True
     self.stop_event = threading.Event()
 
+  def time_series_sort_key(self, ts):
```

Comment:
Nit: This does not need to be a public method on this class. We can make it a private free function.

Reply:
Done.